elements/pl-string-input/pl-string-input.py

```diff
@@ -184,6 +187,10 @@ def parse(element_html, data):
         data['submitted_answers'][name] = None
         return
 
+    if normalize_to_ascii:
+        a_sub = unidecode(a_sub)
```

Comment:
Should we also normalize `data["correct_answers"][xxx]` in `grade()`? Or just a correct answer specified in the HTML?

Reply:
I don't think normalizing correct answers is necessary, personally, because those are supplied by the instructor.  If you as a developer are SPAMing smart quotes all over your questions, it seems like you might have bigger problems...